tavla/app/(admin)/edit/[id]/components/MetaSettings/Organization.tsx

```diff
@@ -1,62 +1,42 @@
 'use client'
-import { Dropdown } from '@entur/dropdown'
+import { Dropdown, NormalizedDropdownItemType } from '@entur/dropdown'
 import { Checkbox } from '@entur/form'
-import { Heading3 } from '@entur/typography'
+import { Heading4 } from '@entur/typography'
 import { FormError } from 'app/(admin)/components/FormError'
-import { useOrganizations } from 'app/(admin)/hooks/useOrganizations'
-import { getFormFeedbackForField } from 'app/(admin)/utils'
-import ClientOnly from 'app/components/NoSSR/ClientOnly'
-import { SubmitButton } from 'components/Form/SubmitButton'
-import { useState, useActionState } from 'react'
-import { TBoardID, TOrganization } from 'types/settings'
-import { moveBoard as moveBoardAction } from './actions'
-import { useToast } from '@entur/alert'
+import { TFormFeedback } from 'app/(admin)/utils'
+import { useState, Dispatch, SetStateAction } from 'react'
+import { TOrganization } from 'types/settings'
 
 function Organization({
-    bid,
     organization,
+    organizations,
+    selectedOrganization,
+    setSelectedOrganization,
+    feedback,
 }: {
-    bid: TBoardID
     organization?: TOrganization
+    organizations: () => NormalizedDropdownItemType<TOrganization>[]
+    selectedOrganization: NormalizedDropdownItemType<TOrganization> | null
+    setSelectedOrganization: Dispatch<
+        SetStateAction<NormalizedDropdownItemType<TOrganization> | null>
+    >
+    feedback?: TFormFeedback
```

Comment:
Kan vi se på å heller flytte disse statsa ned til denne komponenten i stedet for å sende de ned som props?
 
Så har vi heller en useState i parent-komponenten som holder track på om noe endres eller ikke fra child-komponenten. Se docs her: https://react.dev/learn/you-might-not-need-an-effect#notifying-parent-components-about-state-changes